lib/collections/teams.js

```diff
@@ -190,6 +203,9 @@ Meteor.methods({
     else if (password.trim() !== team.password) {
       throw new Meteor.Error(400, 'Incorrect password!');
     }
+    else if (team.inPerson && !user.inPersonAllowed) {
```

Comment:
Since no mixing of in-person/remote is allowed on teams, should there also be a check for the opposite condition (!team.inPerson && user.inPersonAllowed)?

Reply:
Good question, and I'm worried this could lead to confusion for users too. But basically, regardless of what your ticket type is, you are expected to play based on what the _team_ setting is. So theoretically, everyone on the team could buy "in-person" tickets but then set their team to be "virtual" and they'd all be expected to play virtually. (ie the converse of this check is allowed.)

Reply:
Ah okay so that's fine how it is. It's just that virtual teams don't have an all virtual ticket code requirement, but in-person teams do require all in-person tickets. Thanks for clarifying!